apps/aevm/src/aevm_eeevm.erl

```diff
@@ -425,6 +425,13 @@ loop(CP, StateIn) ->
                     <<Val:256/integer-unsigned>> = Hash,
                     State4 = push(Val, State3),
                     next_instruction(CP, State, State4);
+                %% 2f: Non-standard Env information
+                ?CREATOR ->
+                    %% 0x2f Creator δ=0 α=1
+                    %% Get address of contract creator.
```

Comment:
So basically the intention is exposing the `owner` field [already stored in the contract](https://github.com/aeternity/protocol/blame/c638d7728380bfdf85968dba1d22672f49664727/serializations.md#L387)?

Reply:
Yes, and this instruction was already advertised in the Sophia documentation, but we forgot to implement it ;-)